apps/ledger-live-desktop/release-notes.json

```diff
@@ -1,6 +1 @@
-[
-  {
-    "tag_name": "2.43.0",
-    "body": "\nWe’re constantly adding new integrations and working on performance improvements to make Ledger Live a world-class experience. Here is what’s new in this release.\n\n### 🚀 Features\nWe’re excited to announce that Ledger Live is launching support for Cardano, one of the biggest cryptocurrencies based on its total market value. Now you can send and receive Cardano’s native token, ADA. \n\n### 🐛 Fixes\nWe’ve done some bug fixes and made a few small but important changes behind the curtain.\n"
-  }
-]
\ No newline at end of file
+[]
```

Comment:
@elbywan do you know why I get this diff? I didn't touch this file

Reply:
👋 it's autogenerated during the build but I don't know the details, @valpinkman knows more about it